packages/wallet-service/src/api/wallet.ts

```diff
@@ -351,8 +351,7 @@ export const load: APIGatewayProxyHandler = middy(async (event) => {
      */
     await invokeLoadWalletAsync(xpubkeyStr, maxGap);
   } catch (e) {
-    logger.error('Error on lambda wallet invoke', e);
-
+    logger.error(e);
```

Comment:
Why did you change this?

Reply:
That's because Winston as it is configured ignores the second parameter

Created an issue to refactor this: https://github.com/HathorNetwork/hathor-wallet-service/issues/165